src/main/drivers/sound_beeper.c

```diff
@@ -33,6 +38,22 @@ static bool beeperInverted = false;
 static uint16_t beeperFrequency = 0;
 #endif
 
+#ifdef BEEPER
+void systemBeepDshot(void)
+{
+#ifdef USE_DSHOT
+    if (!ARMING_FLAG(ARMED)) {
```

Comment:
Is there a difference in behavior for `!ARMING_FLAG(ARMED)` and `!areMotorsRunning()`?

Reply:
@fujin: Yes, `!areMotorsRunning()` during testing was failing from aioracerf3's size requirements.  The macro let it pass.

Reply:
Using `!ARMING_FLAG(ARMED)` is wrong here - motors can also be running when in motor test mode, `areMotorsRunning()` was specifically added to cater for this case as well.

Reply:
Also, this should check to make sure that Dshot beeping is enabled. And it simply breaks all beeping when the motors are running.